### PR TITLE
replacing text cleanup with standard library

### LINKED ISSUE #88: Use clean-text
### Feature Description

`scraper_utils.str_simplified()` was implemented to clean up scraped text. There is a package [clean-text](https://pypi.org/project/clean-text/) that is specialized for exactly that purpose.

### Use Case

Use `clean-text` to clean up text. e.g., Remove repetitive whitespace.

### Solution

Remove `scraper_utils.str_simplified()`. Replace call sites `str_simplified(text)` with something like `cleantext.clean(text, ...)`. Be sure to look at documentation and set arguments as appropriate.

### Alternatives

Check if there is another package that may be better to use instead of `clean-text`.

### PR DESCRIPTION
**Pull request by Shak2000**
### Link to Relevant Issue

This pull request resolves #88 

### Description of Changes

Replace the `str_simplified` function with `clean-text`.

### CODE REVIEW

**Review by dphoria — APPROVED** [review]
Thank you Shak! Looks good to me.

I know you mentioned this in your [comment in the issue](https://github.com/CouncilDataProject/cdp-scrapers/issues/88#issuecomment-1133526707); are you planning to remove the existing `str_simplified` in this PR or in another?

**Review by JacksonMaxfield — APPROVED** [review]
Nice! Thanks for fixing!

**Review by JacksonMaxfield — CHANGES_REQUESTED** [review]
Wait. Can you please add the `clean-text` package to the library dependencies. It should be added with a lower bound version pin or similar. I prefer `~=`.

**Comment by JacksonMaxfield** [comment]
Do tests no run on PR @dphoria ? odd.

**Comment by dphoria** [comment]
> Wait. Can you please add the `clean-text` package to the library dependencies. It should be added with a lower bound version pin or similar. I prefer `~=`.

Good point; missed that. Thank you Jackson.

**Comment by dphoria** [comment]
> Do tests not run on PR @dphoria ? odd.

No we have an open issue for that. Test got added to the repo by Shak (thank you Shak) abs then he couldn't quite figure out how to run it in our pipeline. Then I just never got to it LOL.

**Comment by Shak2000** [comment]
I have added the necessary dependency.

File: setup.py [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "defusedxml~=0.7.1",
     "pytz~=2021.1",
     "requests~=2.25",
+    "clean-text~=0.6.0"
```
Comment (dphoria) [inline setup.py:35]: Last nitpick comment, Shak. Like the rest of the lists on this file, can we keep the comma `,` here? i.e. `"clean-text~=0.6.0",`

**Review by dphoria — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (Shak2000) [reply setup.py]: Done!

**Review by dphoria — APPROVED** [review]
Thank you Shak. :clap:

**Pull request merged**